content/docs/start/visualization.md

```diff
@@ -0,0 +1,101 @@
+# Visualization with DVC Plots
+
+As we discussed in [experiments] section, DVC versions the experiments by
+associating parameters and metrics to them. On top of that, DVC can generate
+plots and tracks plot images you generated from the code. As in other aspects of
+the experiments, DVC is flexible and provides more than one option to fit
+visualization to your workflow:
+
```

Comment:
Not sure what order makes the most sense here 🤔 . Not a blocker, but worth considering.
* Data series plots are the original and standard. They encourage best practices of saving data over visualizations of that data. I would say they are the most complex to understand, but they might also be helpful to understanding how other plots types are handled.
* Images are the most flexible and maybe most familiar/easiest for people to grasp.
* DVCLive requires the least amount of user code and resembles the most common plot provided in frameworks like mlflow.